syntax = "proto3";

import "google/protobuf/empty.proto";

option go_package = "gen/go/accountant";

service AccountantsService {
  rpc CreateXLSXAccountant (google.protobuf.Empty) returns (CreateXLSXAccountantsResponse);
}

message CreateXLSXAccountantsResponse {
  string xlsx_path = 1;
}
